bundles/AdminBundle/config/security_services.yaml

```diff
@@ -21,39 +22,13 @@ services:
             - Pimcore\Bundle\AdminBundle\Security\Hasher\AdminPasswordHasher
 
     # Authenticators for handling admin login and session authentications
-    Pimcore\Bundle\AdminBundle\Security\Authenticator\AdminLoginAuthenticator:
-        public: false
-        calls:
-            - [ setLogger, [ '@logger' ] ]
-        tags:
-            - { name: monolog.logger, channel: security }
-
     Pimcore\Bundle\AdminBundle\Security\Authenticator\AdminTokenAuthenticator:
         public: false
         calls:
             - [ setLogger, [ '@logger' ] ]
         tags:
             - { name: monolog.logger, channel: security }
 
-    Pimcore\Bundle\AdminBundle\Security\Authenticator\AdminSessionAuthenticator:
-        public: false
-        calls:
-            - [ setLogger, [ '@logger' ] ]
-        tags:
-            - { name: monolog.logger, channel: security }
-
-    pimcore.security.authenticator.admin_pre_auth:
```

Comment:
so [authenticate against admin session](https://pimcore.com/docs/pimcore/current/Development_Documentation/Development_Tools_and_Details/Security_Authentication/Authenticate_Against_Admin_Session.html) doesn't work anymore?

Reply:
true, never needed that.... I would rather create the firewall similar to how pimcore's firewall is. Is that actually a used feature?

Reply:
Honestly, I'm not sure if and how this is being used. Maybe for testing purposes, when editors wants to test some sections on the frontend(which requires authentication) while changing data in the backend. Anyway, if we decide to kick this feature then we need to trigger the deprecation in 10.6.

@brusch do you have any suggestions?

Reply:
@dvesh3 @dpfaffenbauer let's kick that feature - thanks! 😊

Reply:
@brusch ok, I'll deprecate it in 10.6 then

Reply:
Thanks a lot! 👍